packages/bot-web-ui/src/components/load-modal/__tests__/local-footer.spec.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { mock_ws } from 'Utils/mock';
+import RootStore from 'Stores/root-store';
+import { DBotStoreProvider, mockDBotStore } from 'Stores/useDBotStore';
+import LocalFooter from '../local-footer';
+import { isMobile } from '@deriv/shared';
+import DashboardStore from 'Stores/dashboard-store';
+import LoadModalStore from 'Stores/load-modal-store';
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(),
+}));
+
+jest.mock('@deriv/bot-skeleton/src/scratch/blockly', () => jest.fn());
+jest.mock('@deriv/bot-skeleton/src/scratch/dbot', () => ({
+    saveRecentWorkspace: jest.fn(),
+    unHighlightAllBlocks: jest.fn(),
+}));
+
+describe('LocalFooter', () => {
+    let wrapper: ({ children }: { children: JSX.Element }) => JSX.Element, mock_DBot_store: RootStore | undefined;
+
+    beforeAll(() => {
+        const mock_store = mockStore({});
+        mock_DBot_store = mockDBotStore(mock_store, mock_ws);
+        mock_DBot_store = {
```

Comment:
@vinu-deriv instead of mocking the store, is it possible to just let it run as it is and test if the observables are changing in the expected manner?

Reply:
Sure Bhai!